profile/main.go

```diff
@@ -52,6 +52,7 @@ type Res struct {
 	TwitterId   string `json:"twitter_id"`
 	InstagramId string `json:"instagram_id"`
 	Website     string `json:"website"`
+	Dob         string `json:dob`
 }
 
 type Diff struct {
```

Comment:
Diff looks like the struct which is going in firestore, what do we use Res for? @RitikJaiswal75

Reply:
I think its the response obj, let me know if its correct

Reply:
this struct is used in case you make changes in your profile service. So the diff struct is used which represents the difference in the profile data. this also gets saved in the db and replaces the original data post approval from dashboard.

Reply:
Got it